src/main/java/com/slimebot/alerts/spotify/SpotifyListener.java

```diff
@@ -120,4 +120,14 @@ private void broadcast(String format, Function<SpotifyNotificationConfig, Option
 	public static Logger getLogger() {
 		return logger;
 	}
+
+	private void fetchToken(SpotifyApi api){
```

Comment:
Why do you have the SpotifyApi as parameter? It is available as field in the class